B/Bullet/build_tarballs.jl

```diff
@@ -0,0 +1,58 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Bullet"
```

Comment:
Should this be called `Bullet_Physics` or `Bullet_Physics_SDK`?  Remember that if you change this name you also must rename the directory accordingly

Reply:
They tend to style it just "Bullet". "Bullet Physics" sorta makes it sound like physics just for bullets, as if someone wrote a library just for that. Which wouldn't be far-fetched, knowing Julia's community.

Reply:
Nevermind, there are other "Bullet Engines" out there, I'll go for "Bullet_Physics_SDK".